### DISCUSSION TITLE
关于黑名单和白名单的理解

### DISCUSSION THREAD

**Discussion by ixmu** [pymumu/smartdns#1464] [Q&A] [answered]
-whitelist-ip 理解为，仅接受白名单里面的ip，其他的抛弃
-blacklist-ip 理解为，抛弃掉黑名单中的ip，其他的接受

请问这样理解是否正确？

**Comment by PikuZheng** [comment, marked as answer]
当上游返回的结果**之一**在blacklist中时，丢弃**整个上游的结果**（即使其他ip不在blacklist中）
whitelist我自己不用，原理上应该是只保留whitelist范围内的ip
